contrib/bloop/src/mill/contrib/bloop/BloopImpl.scala

```diff
@@ -112,7 +112,8 @@ class BloopImpl(ev: () => Evaluator, wd: os.Path) extends ExternalModule { outer
   protected def computeModules: Seq[JavaModule] = {
     val eval = ev()
     if (eval != null)
-      transitiveModules(eval.rootModule).collect { case jm: JavaModule => jm }
+      ModuleUtils.transitiveModules(eval.rootModule, accept = _.isInstanceOf[JavaModule])
+        .collect { case jm: JavaModule => jm }
```

Comment:
@lolgab I woulder why this is not the equivalent of the replaced call. The failed tests suggest there is a difference. Any idea?

Reply:
Nevermind, I found the issue.